### ISSUE TITLE
Enable pushing docs to gh-pages branch as part of build

### ISSUE THREAD

**Issue by joelea** [palantir/atlasdb#382]

**Comment by jboreiko** [comment]
Blocked on the addition of palantir-atlasdb-circleci to the palantir organization.

**Comment by jboreiko** [comment]
Fixed by allowing more open permission for pushing to gh-pages, filing separate ticket for locking them back down once a new user is given permission for circleci to use.
